src/freenas/etc/periodic.conf

```diff
@@ -27,3 +27,9 @@ security_show_success="NO"
 weekly_locate_enable="NO"
 weekly_whatis_enable="NO"
 monthly_show_success="NO"
+daily_output="/dev/null"
+daily_status_security_output="/dev/null"
+weekly_output="/dev/null"
+weekly_status_security_output="/dev/null"
+monthly_output="/dev/null"
+monthly_status_security_output="/dev/null"
```

Comment:
That means periodic will still run?

Do we need that? Seems it would be more clear if we remove this failed and stop calling periodic in crontab?